external/fv3fit/fv3fit/emulation/transforms/factories.py

```diff
@@ -72,62 +71,59 @@ def interp(x: tf.Tensor) -> tf.Tensor:
 class ConditionallyScaled(TransformFactory):
     """Conditionally scaled transformation
 
-    Scales the output-input difference of ``field`` by conditional standard
-    deviation and mean::
+    Scales ``source`` by conditional standard deviation and mean::
 
-                  d field - E[d field|on]
+                  source - E[source|on]
         to =  --------------------------------
-               max[Std[d field|on], min_scale]
+               max[Std[source|on], min_scale]
 
     Attributes:
         to: name of the transformed variable.
         condition_on: the variable to condition on
         bins: the number of bins
-        field: the prognostic variable spec. The difference between input and
-            output data (``d field``) is normalized.
+        source: The variable to be normalized.
         min_scale: the minimium scale to normalize by. Used when the scale might
             be 0.
-        filter_magnitude: any values with |to-field| < filter_magnitude are removed
-            from the standard deviation/mean calculation.
+        filter_magnitude: if provided, any values with
```

Comment:
Consider renaming to signal that it only affects the scaling fit instead of all outputs? e.g., `fit_filter_magnitude`